<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>merge&amp;date</string>
  </property>
  <property name="locale">
   <locale language="Ukrainian" country="Ukraine"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>20</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>130</width>
      <height>20</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>Введіть бажану дату</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>85</width>
      <height>25</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Об'єднати файли</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Виставити дату</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>5</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Введіть дату:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>85</width>
      <height>25</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Прибрати все з файлів</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Виставити прізвища</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>220</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>20</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>130</width>
      <height>20</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>ПІБ перевіряючого</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>20</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>130</width>
      <height>20</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>ПІБ затверджуючого</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Перевірив:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Затвердив:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Привіт))</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
